File: cfgs/java_cfgs_foo/COUNT_OFDIFFERENT_WAYS_EXPRESS_N_SUM_1_3_4.java
Nodes
1;special;;;
2;statement;int[] DP = new int[foo_1 + 1];14;
3;statement;foo_1[0] = foo_2[1] = foo_3[2] = 1;15;
4;statement;foo_1[3] = 2;16;
5;statement;int i = 4;17;
6;conditional;foo_1 <= foo_2;18;
7;statement;foo_1[foo_2] = (foo_3[foo_4 - 1] + foo_5[foo_6 - 3]) + foo_7[foo_8 - 4];19;
8;statement;i++;19;
9;statement;return foo_1[foo_2];20;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;9;
6;8;
7;6;
8;10;
9;

File: cfgs/pythoncfgs/COUNT_OFDIFFERENT_WAYS_EXPRESS_N_SUM_1_3_4.py
Nodes
1;statement;foo_1 = [0 for foo_2 in foo < foo];8;
2;statement;foo_1[0] = foo_2[1] = foo_3[2] = 1;9;
3;statement;foo_1[3] = 2;10;
4;conditional;foo < foo;11;
5;statement;return foo_1[foo_2];13;
6;statement;foo_1[foo_2] = foo_3[foo_4 - 1] + foo_5[foo_6 - 3] + foo_7[foo_8 - 4];12;
7;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;7;
5;4;
6;
